appraisal.lic

```diff
@@ -77,21 +77,17 @@ class Appraisal
     end
     pause 1
     $ORDINALS.each do |ordinal|
-      case bput("get #{ordinal} #{gem_pouch_adjective} #{gem_pouch_noun} from my #{full_pouch_container}", '^You get ', '^What were you referring')
-      when /^You get /
-        case bput("appraise my #{gem_pouch_adjective} #{gem_pouch_noun} quick", 'Roundtime', /You.ll need to open the .* to examine its contents./)
-        when /You.ll need to open the .* to examine its contents./
-          bput("open my #{gem_pouch_adjective} #{gem_pouch_noun}", /You open your .* ./, /That is already open/)
-          bput("appraise my #{gem_pouch_adjective} #{gem_pouch_noun} quick", 'Roundtime')
-          bput("close my #{gem_pouch_adjective} #{gem_pouch_noun}", /You close your/, /That is already closed/)
-        end
-        waitrt?
-        bput("put my #{gem_pouch_adjective} #{gem_pouch_noun} in my #{full_pouch_container}", 'You put')
-        pause 1
-      else
-        break
+    break unless DRCI.get_item?( "#{ordinal} #{gem_pouch_adjective} #{gem_pouch_noun}", full_pouch_container )  
+      case bput("appraise my #{gem_pouch_adjective} #{gem_pouch_noun} quick", 'Roundtime', /You.ll need to open the .* to examine its contents./)
+      when /You.ll need to open the .* to examine its contents./
+        bput("open my #{gem_pouch_adjective} #{gem_pouch_noun}", /You open your .* ./, /That is already open/)
+        bput("appraise my #{gem_pouch_adjective} #{gem_pouch_noun} quick", 'Roundtime')
+        bput("close my #{gem_pouch_adjective} #{gem_pouch_noun}", /You close your/, /That is already closed/)
       end
-      break if DRSkill.getxp('Appraisal') >= 30
+      waitrt?
+      DRCI.put_away_item?( "#{gem_pouch_adjective} #{gem_pouch_noun}", full_pouch_container )
+      pause 1
+    break if DRSkill.getxp('Appraisal') >= 30
```

Comment:
Thanks for the update. 

💅  Minor stylistic request that lines 80-90 be indented two more spaces so that they fall nested wtihin the each loop defined at line 79

Reply:
good call, missed that when I updated things

Reply:
I do not believe lines 81 through 89 require two spaces indented. You are not in a specific unless loop as `break unless ...` is its own loop code line.